spk/tt-rss/Makefile

```diff
@@ -51,13 +54,11 @@ $(STAGING_DIR)/etc/php/conf.d/com.synocommunity.tt-rss.ini:
 .PHONY: tt-rss_extra_install
 tt-rss_extra_install: $(STAGING_DIR)/etc/php/conf.d/com.synocommunity.tt-rss.ini
 	install -m 755 -d $(STAGING_DIR)/var
+	install -m 755 -d $(STAGING_DIR)/var/logs
 	install -m 755 -d $(STAGING_DIR)/app
 	install -m 644 src/app/config $(STAGING_DIR)/app/config
-	install -m 755 src/bin/guess-php-fpm-bin $(STAGING_DIR)/bin/guess-php-fpm-bin
```

Comment:
All of those files were turned into functions defined in the `settings.sh` file.